spec/factories/factories.rb

```diff
@@ -20,7 +20,7 @@
     context 'self'
     lines [42]
     message 'smell warning message'
-    parameters     {}
+    parameters Hash.new
```

Comment:
RuboCop 0.29 doesn't like this (and neither do I :smile:). Perhaps `parameters { {} }` ?